levi/__init__.py

```diff
@@ -96,3 +99,13 @@ def boundary_parser(boundary):
     else:
         raise ValueError(
             f"{boundary} cannot be parsed.  Valid prefixes are <, <=, >, and >=.  Two values can be delimited with a dash.  Valid examples include <10kb and 10kb-5gb.")
+
+def updated_partitions(delta_table: DeltaTable, start_time: Optional[datetime.datetime] = None, end_time: Optional[datetime.datetime] = None) -> dict[str, str]:
```

Comment:
The return type might be wrong here. I think this should have been List[dict[str, str]]